Debug/TraceableMetadataFactory.php

```diff
@@ -32,7 +34,7 @@ public function getAllClassNames(): array
         return $this->metadataFactory->getAllClassNames();
     }
 
-    public function getMetadataForClass(string $className)
+    public function getMetadataForClass(string $className): ClassHierarchyMetadata|MergeableClassMetadata|null
```

Comment:
It looks like the package still supports PHP 7.2 that do not support such syntax. May there other way to fix this deprecation message?

Reply:
I'll check that

Reply:
Just like the deprecation message says, `add an explicit @return annotation to suppress this message`.  The B/C break can be dealt with later.

Reply:
I've added @return annotation